docs/pages/includes/node-logins.mdx

```diff
@@ -0,0 +1,30 @@
+When Teleport's Auth Service receives a request to list Teleport Nodes (e.g., to
+display Nodes in the Web UI or via `tsh ls`), it only returns the Nodes that the
+current user is authorized to access.
```

Comment:
Nit: I would say "see" (or analogue) instead of "access" because you can see the node but still have not permissions to access it if allowed logins don't allow it.
```suggestion
current user is authorized to see.
```